### ISSUE TITLE
Release 8.0.11?

### ISSUE THREAD

**Issue by mjhenkes** [postcss/postcss-custom-properties#180]
Could you cut a release with the upgraded postcss-values-parser dependency? postcss-values-parser version 2.x.x use flatten which has been deprecated and warns on npm install.

**Comment by jonathantneal** [comment]
@mjhenkes, if I bump values-parser by a major version, that makes it a major release.

**Comment by mjhenkes** [comment]
You've already upgraded it:
https://github.com/postcss/postcss-custom-properties/commit/0e5a66df910b65247133e13a685ca5c8cab4e60f

Could you cut the next release? Whether thats major or minor is up to you.

**Comment by tanshio** [comment]
@jonathantneal 

Please upgrade v8.0.11.
postcss-values-parser 2.0.1 contains a bug...

```
margin: calc(var(--gutter) * 0.5);
```

postcss-values-parser 3.x success.

**Comment by EECOLOR** [comment]
For the next release, please also fix this bug: https://github.com/postcss/postcss-custom-properties/pull/182

**Comment by jonathantneal** [comment]
Since breaking changes are already forced for the next update, I’ll be moving the transforms to https://github.com/csstools/postcss-custom-utils — this is because all custom variables already share a lot of similar tooling — duplicated even further to support PostCSS Preset Env.

**Comment by jonathantneal** [comment]
1. **PostCSS Custom Properties 8.0.11** has been released. This is a patch release containing all of the non-breaking fixes.
   - https://github.com/postcss/postcss-custom-properties/releases/tag/8.0.11

2. **PostCSS Custom Properties 9.0.0** has also been released. This is a major release containing the breaking update to PostCSS Values Parser 3 which resolves parsing issues.
   - https://github.com/postcss/postcss-custom-properties/releases/tag/9.0.0
   - _This version has not yet been published at this time because npm is claiming it has already been published, despite this not being so (see https://www.npmjs.com/package/postcss-custom-properties?activeTab=versions)._
   ```
   npm ERR! code E403
   npm ERR! 403 Forbidden - PUT https://registry.npmjs.org/postcss-custom-properties - You cannot publish over the previously published versions: 9.0.0.
   ```

Thank you for being persistent and patient, everyone.

**Comment by mjhenkes** [comment]
Thanks @jonathantneal !

**Comment by jonathantneal** [comment]
**PostCSS Custom Properties 9.0.1** has been released. This release is identical to v9.0.0, only `npm publish` failed to publish v9.0.0 and threw the following error:

```
You cannot publish over the previously published versions: 9.0.0.
```

I did not want this issue to distract me, and so I thoughtfully and impatiently published v9.0.0 as v9.0.1.

Feedback from npm support (https://twitter.com/npm_support/status/1141714702176792578).
